blockchain/system/kip113_test.go

```diff
@@ -93,17 +113,20 @@ func TestAllocKip113(t *testing.T) {
 				Balance: common.Big0,
 			},
 		}
-		backend     = backends.NewSimulatedBackend(alloc)
-		contract, _ = contracts.NewKIP113MockTransactor(KIP113MockAddr, backend)
+		backend               = backends.NewSimulatedBackend(alloc)
+		contractAddr, _, _, _ = contracts.DeployERC1967Proxy(sender, backend, KIP113MockAddr, input)
 	)
+	backend.Commit()
+
+	contract, _ := contracts.NewKIP113MockTransactor(contractAddr, backend)
 
 	contract.Register(sender, nodeId1, pub1, pop1)
 	contract.Register(sender, nodeId2, pub2, pop2)
 	backend.Commit()
 
 	execStorage := make(map[common.Hash]common.Hash)
 	stateDB, _ := backend.BlockChain().State()
-	stateDB.ForEachStorage(KIP113MockAddr, func(key common.Hash, value common.Hash) bool {
+	stateDB.ForEachStorage(contractAddr, func(key common.Hash, value common.Hash) bool {
```

Comment:
KIP113 storage layout is not identified. To ensure that there's no wrong calculation, the removed one should be reverted. Doesn't it?

Reply:
Can I ask what is the `the removed one`?

Reply:
I overlooked that there was manual registration. I found that is manually prepared in L117-118. Sorry for the noise.